docs/src/gui/halui.adoc

```diff
@@ -165,28 +163,28 @@ that is set in the [TRAJ] section of the INI file.
 
 === MDI
 
-* 'halui.mdi-command-<nn>' (bit, in) - halui will try to send the MDI
-  command defined in the INI. <nn> is a  two  digit number  starting  at 00. +
+* 'halui.mdi-command-_<nn>_' (bit, in) - halui will try to send the MDI
+  command defined in the INI. _<nn>_ is a two  digit number  starting  at 00. +
   If the command succeeds then it will place LinuxCNC in the MDI mode and then back to Manual mode. +
   If no [HALUI]MDI_COMMAND variables are set in the  ini  file, no halui.mdi-command-<nn> pins will be
   exported by halui.
 
 === Joint
 
-N = joint number (0 ... num_joints-1) +
+_N_ = joint number (0 ... num_joints-1) +
 Example:
 
-* 'halui.joint.N.select' (bit in) - pin for selecting joint N
-* 'halui.joint.N.is-selected' (bit out) - status pin that joint N is selected
-* 'halui.joint.N.has-fault' (bit out) - status pin telling that joint N has a fault
-* 'halui.joint.N.home' (bit in) - pin for homing joint N
-* 'halui.joint.N.is-homed' (bit out) - status pin telling that joint N is homed
-* 'halui.joint.N.on-hard-max-limit' (bit out) - status pin telling that joint N  is  on  the  positive  hardware limit
-* 'halui.joint.N.on-hard-min-limit' (bit out) - status  pin  telling  that  joint  N is on the negative hardware limit
-* 'halui.joint.N.on-soft-max-limit' (bit out) - status pin telling that joint N  is  on  the  positive  software limit
-* 'halui.joint.N.on-soft-min-limit' (bit out) - status  pin  telling  that  joint  N is on the negative software limit
-* 'halui.joint.N.override-limits' (bit out) - status pin telling that joint N's limits are  temporarily  overridden
-* 'halui.joint.N.unhome' (bit in) - pin for unhoming joint N
+* 'halui.joint._N_.select' (bit in) - pin for selecting joint _N_
```

Comment:
Furthermore this changes does absolutely no effect because `' '` already makes it italic.

Reply:
Nope. https://docs.asciidoctor.org/asciidoc/latest/text/italic/

Reply:
'not italic', __italic_\_

Reply:
And yes, I would like to remove 90% of the "'"s in the documentation.

Reply:
Uh, seems like I just learned something - https://linuxcnc.org/docs/devel/html/gui/halui.html shows these lines as italic, indeed. Hm.

Reply:
So you are not building the docs to verify your changes?

Reply:
I look at them in the previewer.  The .N. -> ._N_. change was semi-automated.

Reply:
There is something else to it. The _N\_ indicates that this N is variable, since being slanted is often seen to indicate such variability. I would not want to take those semantics away, even if invisible. If the outer highlighting would be bold, this would have worked just fine. Also, slanting something within something slanted could "undo" the slanting, which then depends on some future asciidoc rendering. So, I suggest to keep those inner _N\_s.